src/embind/embind.js

```diff
@@ -2533,7 +2533,7 @@ var LibraryEmbind = {
     return name;
   },
 
-  _embind_register_smart_ptr__sig: 'vpppipppppppp',
+  _embind_register_smart_ptr__sig: 'v',
```

Comment:
this seems wrong? Aren't there a bunch of parameters?

Reply:
Very interesting.  Good catch.  I found the issue where clang seemed to drop the signature for this function (seems related to a forward-declared enum as one of the params).